process/block/metrics.go

```diff
@@ -226,7 +228,38 @@ func saveRoundInfoInElastic(
 			Timestamp:        time.Duration(header.GetTimeStamp() - ((currentBlockRound - i) * roundDuration)),
 		}
 
-		go elasticIndexer.SaveRoundInfo(roundInfo)
+		go indexerHandler.SaveRoundInfo(roundInfo)
+	}
+}
+
+func indexValidatorsRating(
+	indexerHandler indexer.Indexer,
+	valStatProc process.ValidatorStatisticsProcessor,
+	metaBlock data.HeaderHandler,
+) {
+
+	latestHash, err := valStatProc.RootHash()
+	if err != nil {
+		return
+	}
+
+	validators, err := valStatProc.GetValidatorInfoForRootHash(latestHash)
+	if err != nil {
+		return
+	}
+
+	for shardID, validatorInfosInShard := range validators {
```

Comment:
please add a todo to use the validatorInfoProvider GetLatestValidators() method in order to have the rating already computed and not divide by 1.000.000 this is actually the maxRating

Reply:
done.

Reply:
ok